storage/storage_test.go

```diff
@@ -68,23 +68,23 @@ func newStore(t *testing.T) storage.Store {
 	}
 	run := filepath.Join(wd, "run")
 	root := filepath.Join(wd, "root")
-	uidmap := []idtools.IDMap{{
+	Transport.SetDefaultUIDMap([]idtools.IDMap{{
 		ContainerID: 0,
 		HostID:      os.Getuid(),
 		Size:        1,
-	}}
-	gidmap := []idtools.IDMap{{
+	}})
+	Transport.SetDefaultGIDMap([]idtools.IDMap{{
 		ContainerID: 0,
 		HostID:      os.Getgid(),
 		Size:        1,
-	}}
+	}})
 	store, err := storage.GetStore(storage.StoreOptions{
 		RunRoot:            run,
 		GraphRoot:          root,
 		GraphDriverName:    "vfs",
 		GraphDriverOptions: []string{},
```

Comment:
Consumers of libraries usually (or at least the ones I know about) don’t vendor in tests, so this should not affect them at all.